sys-kernel/installkernel-systemd-boot/files/installkernel-0.0.1.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+create_kernel_dir() {
+	# logic flow taken from kernel-install itself
+	local machine_id kernel_dir
+
+	if [[ -d /efi/Default ]] || [[ -d /boot/Default ]] || [[ -d /boot/efi/Default ]]; then
+		machine_id="Default"
+	elif [[ -f /etc/machine-id ]]; then
+		read machine_id < /etc/machine-id
+	else
+		machine_id="Default"
+	fi
+
+	# now we know id, let's determine destination
+	# follow kernel-install logic of determining ENTRY_DIR_ABS
+	if [[ -d /efi/loader/entries ]] || [[ -d /efi/${machine_id} ]]; then
+		kernel_dir="/efi/${machine_id}"
+	elif [[ -d /boot/loader/entries ]] || [[ -d /boot/${machine_id} ]]; then
+		kernel_dir="/boot/${machine_id}"
+	elif [[ -d /boot/efi/loader/entries ]] || [[ -d /boot/efi/${machine_id} ]]; then
+		kernel_dir="/boot/efi/${machine_id}"
+	elif mountpoint -q /efi; then
+		kernel_dir="/efi/${machine_id}"
+	elif mountpoint -q /boot/efi; then
+		kernel_dir="/boot/efi/${machine_id}"
+	else
+		kernel_dir="/boot/${machine_id}"
+	fi
```

Comment:
I suppose the different /boot/efi vs /efi order comes from kernel-install itself?